.gitignore

```diff
@@ -9,4 +9,9 @@ Cargo.lock
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-.venv
\ No newline at end of file
+.venv
+
+.github
+.cargo
```

Comment:
We should not ignore these. .github needs to be checked in for things like our actions, templates, dependabot, etc. Their configuration goes there. Same for .cargo. .cargo stores cargo configuration. That configuration is actually necessary right now for tests to run. See CONTRIBUTING.md.

Reply:
This was supposed to capture .cargo and .github

Reply:
I'll remove those lines in the next commit